src/Microsoft.AspNetCore.Razor.Language/DefaultRazorProjectEngine.cs

```diff
@@ -66,12 +66,25 @@ protected override RazorCodeDocument CreateCodeDocumentCore(RazorProjectItem pro
 
             var importFeature = GetRequiredFeature<IImportProjectFeature>();
             var importItems = importFeature.GetImports(projectItem);
-            var importSourceDocuments = GetImportSourceDocuments(importItems);
+            var importSourceDocuments = GetImportSourceDocuments(importItems, suppressExceptions: false);
```

Comment:
What's the change? was this a bug?

Reply:
Nah I can revert this. `suppressExceptions` is a default param with value false anyway.